packages/preferences/src/components/preferences-modal-tabs/style.scss

```diff
@@ -1,12 +1,15 @@
 $vertical-tabs-width: 160px;
 
 .preferences__tabs-tablist {
-	position: absolute;
+	position: absolute !important;
```

Comment:
Would be nice if we could avoid these `!important` rules.

Reply:
Agreed - unfortunately, emotion is setting its styles with a higher precedence than those set by consumers, so it is hard to override without doing this. I can try using specificity tricks or something like that but I'm not sure they'll work. I wonder if I'm missing something though, as these overrides seem common enough for the team to have realized this style precedence issue before? cc @mirka as she was also surprises during a pairing session the other day.

Reply:
We do have control over that `position: relative` that Emotion sets though, right? I think @mirka suggested using `isolation: isolate` instead of `position: relative` and if I understand correctly, it should fix that issue and also remove the need to use `!important` here.

Reply:
Yes, that's true for this instance and it's definitely a cleaner / more explicit implementation, so I'll go for that 👍 

The other instance remains though, so I still wonder about the precedence issue. Did you know about it beforehand? Has there been some prior discussion of this I can read?

Reply:
I guess you're talking about `content: none !Important`? 

Won't that also be resolved once we use a shared animation backdrop like we do for `ToggleGroupControl`? Then for the preferences modal tabs, we can just hide the backdrop.

Reply:
Yes, that's also correct. I still wonder about the precedence issue though.

Reply:
Well, getting rid of Emotion will rid us of those issues 😉

Reply:
Need to stick with `!important` unfortunately. Not an issue though, since it makes no sense to expect further overrides. Can remove once we migrate off Emotion.